databroker/tests/test_broker.py

```diff
@@ -173,14 +173,6 @@ def test_get_events_filtering_field(db, RE):
 
 
 @py3
-def test_deprecated_api(db, RE):
-    RE.subscribe(db.insert)
-    uid, = RE(count([det]))
-    h, = db(uid=uid)
-    assert list(db.get_events(h))
```

Comment:
Look at the previous 2 commits, I removed `fetch_events` and `find_headers` in 2 steps.  This test now covers existing functionality that is tested else where (without the misleading name).